src/Core/src/Platform/Windows/ContentPanel.cs

```diff
@@ -162,7 +163,7 @@ void UpdateBorder(IShape? strokeShape)
 			UpdateClip(strokeShape, width, height);
 		}
 
-		void UpdateClip(IShape? borderShape, double width, double height)
+		async void UpdateClip(IShape? borderShape, double width, double height)
```

Comment:
Could avoid async void? Because error propagation and also the call stack may not accurately represent the execution flow at the time the exception was thrown, complicating the process of identifying and fixing bugs. 

Return a Task and rename the method to `UpdateClipAsync`.

Reply:
@jsuarezruiz, I have renamed the method to `UpdateClipAsync` and changed its return type to Task. Could you please check once and let me know if you have any concerns?